### ISSUE TITLE
Default values do not get applied as expexted

### ISSUE THREAD

**Issue by maverick74** [aqt/webext-popup#43]
On MSWindows,

On **Workarounds** the "add or substract pixels..." is not working

Same thing goes for positioning on **Popup Rules**' defaults.

Per domain/url rules positioning work well, however.

(I can be missing something as this was seen in a bit of a hurry...)

**Comment by aqt** [comment]
I'm having the same issue, but activating the "Force position/size" workaround fixes it for me.
Do you have this enabled?

If you recall #39 it was added to address issues when enabling the `privacy.resistFingerprinting` setting, but I have it disabled and am still affected, might be the cause of another change in recent Firefox versions?

**Comment by maverick74** [comment]
Yes, i do.

These are my current settings,
![popup](https://user-images.githubusercontent.com/14912461/69428187-8b96f300-0d28-11ea-80fe-e459ab3e2fb5.jpg)


 but i've tried different combinations and only when setting window position and dimensions on the "per domain/URL fields" i am getting it to position correctly...

**Comment by aqt** [comment]
Thank you, it would appear there are two issues here:

1) The first one, which you reported, is actually expected behavior.
I will change this and release an update.

Rules do not inherit the values from the defaults, the default values are only used if there are no rules matching. All or nothing. Before the introduction of domain/url specificity this was not the case, but _I can not remember why I changed it_.

2) The second problem is what I experienced.

Position and size do not take effect unless I enable the workaround.
This would logically have been caused by a Firefox update, I have not made any configuration or code changes. I don't see any update in their documentation, so perhaps it is simply a bug and will be fixed in an update.
Thus I won't troubleshoot that issue further for now.
